app-accessibility/brltty/brltty-5.2-r1.ebuild

```diff
@@ -166,18 +172,18 @@ src_install() {
 	mv "${D}"/${libdir}/*.a "${D}"/usr/${libdir}/
 	gen_usr_ldscript libbrlapi.so
 
-	cd Documents
+	cd Documents || die
 	mv Manual-BRLTTY/English/BRLTTY.txt BRLTTY-en.txt
 	mv Manual-BRLTTY/French/BRLTTY.txt BRLTTY-fr.txt
 	mv Manual-BrlAPI/English/BrlAPI.txt BrlAPI-en.txt
 	dodoc CONTRIBUTORS ChangeLog HISTORY README* TODO BRLTTY-*.txt
 	dohtml -r Manual-BRLTTY
 	if use doc; then
-		dohtml -r Manual-BrlAPI
 		dodoc BrlAPI-*.txt
+		einstalldocs
 	fi
 
-	keepdir /var/lib/brlapi
+	keepdir /var/lib/BrlAPI
 	rmdir "${D}/run/brltty"
```

Comment:
missing dies

Reply:
Didn't know you needed a die on `keepdir` and other ebuild builtins.

Reply:
keepdir doesn't need die but rmdir probably needs.

Reply:
Ah. Will fix it.